src/main.js

```diff
@@ -61,6 +62,32 @@ let load = loader({
     }),
   },
 
+  reference: {
```

Comment:
Why do we need this level of indirection? `reference()` is not asynchronous... and even if it was, we could just make the `setup` for `docs` component async and it would be fine...